modules/commons/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I checked the code of this module. Sorry if I didn't make it clear last time, but:
1) I still don't see the value of sharing it between Cassandra and JDBC-based containers.
2) If you really want to do this change, I would rather split this PR into two, one change at a time. 
3) previously mentioned naming issues

Reply:
Ok, maybe I didn't clear out enough why I did this in the first place.

I brought DatabaseDelegate approach not just because I wanted to, but because Cassandra script initialization requires the same code that lies in ScriptUtils. ScriptUtils is currently tied up with SQL and JDBC, but in real life database scripts can be different languages and use different databases, and JDBC is not always the answer. Just like with CQL.
If I didn't bring new abstraction, then I would be forced to duplicate massive amount of code and only change java.sql.Connection argument to com.datastax.driver.core.Session. This code duplication is not just bad - it's terrible. 

So I untied script parsing and execution from concrete database implementation by bringing DatabaseDelegate abstraction.